### PR TITLE
Enable additional-serialization-bindings by default, #22169

### PR DESCRIPTION
**Pull request by patriknw**
* and thereby it was also possibe to simplify the documentation
  in remoting

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #22169

**Comment by patriknw** [comment]
The failure was unrelated. Fixed in https://github.com/akka/akka/pull/22215

**Comment by patriknw** [comment]
PLS BUILD

File: akka-docs/rst/scala/remoting.rst [hunk 1]
```diff
@@ -287,47 +287,21 @@ For more information please see :ref:`serialization-scala`.
...
 .. note:: 
-  When using the new remoting implementation (codename Artery), Akka does not use Java Serialization for any of it's internal messages.  
+  Akka does not use Java Serialization for any of it's internal messages.
   It is highly encouraged to disable java serialization, so please plan to do so at the earliest possibility you have in your project.
 
   One may think that network bandwidth and latency limit the performance of remote messaging, but serialization is a more typical bottleneck.
-
-For user messages, the default serializer, implemented using Java serialization, remains available and enabled in Artery.
+  
+For user messages, the default serializer, implemented using Java serialization, remains available and enabled by default.
```
Comment (johanandren) [inline akka-docs/rst/scala/remoting.rst]: I think all commas in this sentence can be removed.

**Review by johanandren — APPROVED** [review]
LGTM. I would wish we could make java serialization opt in rather than on by default, and enabling it a very conscious decision by the user, but I guess that may be to much of a breaking change?

**Comment by patriknw** [comment]
baby steps

**Comment by patriknw** [comment]
I'll remove the commas in another commit which I have to amend anyway

**Pull request merged**